docs/open-telemetry.asciidoc

```diff
@@ -26,10 +28,10 @@ business KPIs and technical components with the {stack}.
 [[open-telemetry-native]]
 ==== APM Server native support of OpenTelemetry protocol
 
-IMPORTANT: The https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticexporter#legacy-opentelemetry-collector-exporter-for-elastic[OpenTelemetry Collector exporter for Elastic]
-was deprecated in 7.13 and replaced by the native support of the OpenTelemetry Line Protocol in
-Elastic {observability} (OTLP). To learn more, see
-https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticexporter#migration[migration].
+****
+Starting in {stack} version 8.3,
```

Comment:
We've had native support for OTLP/gRPC since 7.13, it's only OTLP/HTTP that was added in 8.3. Am I misreading this?

Reply:
If you're "misreading it," then others might too. I'll make it clearer that only OTLP/HTTP is new.

Reply:
Maybe I just misunderstood the purpose of this callout. Is it like a "what's new" type of thing? Slightly concerned that new users will take the statement to mean we support OTLP/HTTP (only) as of 8.3, and the non-mention of OTLP/gRPC may suggest that we don't support that.

Reply:
Yes—"what's new". That's how I interpreted the first bullet point of the referenced issue. I'm all for just removing this and letting the changelog and official "Observability what's new" do the talking.

Reply:
I think that might be best. My suggestion is:
 - in the 8.3.0 "Observability what's new", mention that we now support OTLP/HTTP as well as OTLP/gRPC
 - don't specifically highlight that we added OTLP/HTTP in 8.3 here
 - remove lack of OTLP/HTTP from limitations
 - remove the "IMPORTANT: ..." bit about the old opentelemetry-collector-contrib and introduction of native OTLP support in 7.13. 7.13 was quite a while ago, and the [opentelemetry-collector-contrib exporter docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticexporter) mention that it is deprecated and point here. I don't think we need to continue drawing attention to the old deprecated thing.

@simitt @cyrille-leclerc any concerns?

Reply:
Andrew's suggestions sound good to me; 
the old exporter is only deprecated and not removed, so I am on the fence on mentioning this in the docs, but I am ok with only referencing them directly from the opentelemetry-collector-contrib-exporter as suggested.

Reply:
OK, sounds good. Thanks @bmorelli25!

Reply:
Thanks for your help!